CONTRIBUTING.md

```diff
@@ -0,0 +1,138 @@
+
+```
+flutter clean
+flutter format .
+flutter analyze
+flutter test
+```
+It's ***strongly*** recommended that you enabled "format on save" in your preferred IDE for Flutter (dart) code files.
+
+When contributing to this repository, please first discuss the change you wish to make via slack, issue, or email or any other method with the owners of this repository before making a change.
+
+**Pro tips**
+
+🔥 If your committing changes to the docs ***ONLY*** add `[skip ci]` to **all** of your commit messages to save build time as we are currently burning build minutes 😬. Also prefix your PR title with `Docs: ` so that its obvious to the project team what the PR is.
+
+📱 Pick a device with a low resolution to test on and make sure there is no overflow errors.
+
+🎨 Check against the designs in Zeplin.
+
+☑️ Only open up a PR once you feel you are finished.  You can request a review from `Flutter devs` get reviews
+
+🤗 If your unsure about something ask for help.  We are really friendly!
+
+
+Currently the project is being primarily co-ordinated via the GDG Melbourne slack group. You can join GDG Melbourne's slack via [this invite](http://bit.ly/join_gdgslack). DM `@luke.sleeman` on GDG Melbourne slack to be invited to the channel being used for co-ordination.
+
+If you have any questions about why things are done a certain way in the app, check out the [Architecture Decision Records](docs/Architecture) - they should be able to provide some background on why things are done a specific way.
+
+### Pull Request Process
+
+## Method 1 - If you are part of the team
+- Please make sure the work you are doing is described in an existing Github issue.
+- Develop your work in a branch. Branch names should be prefixed with `feature`, `tech`, `docs` or `bugfix` depending on what is being done in them AND should contain the number of the matching Github issue. eg: `feature/142-update-homescreen`
+- Open a pull request onto master. Be sure to include a detailed description in your pull request per the PR template. Other contributors will comment on your code or approve it
+- Please be sure to note the build status of your PR reported by the Codemagic CI
+- Once your pull request has been approved, please merge into master.
+
+## Method 2 - If you are not yet part of the team
+- If you are not a part of the team, you can still work on the code by clicking on the `Fork` option in this page.
+- Once you have forked to your own github account,  you will be able to check out the fork and work on it.
+- To create a pull request, you would have to push your local changes back to your git accounts fork. 
+- You would then find the option to `Compare and Pull Request`. 
+- Please follow the instructions above on how to submit a pull request. 
+- Once the pull request is reviewed and approved by the team, they would be able to merge this with the main branch. 
+
+⬇️
+
+## The Development process
```

Comment:
Wondering if this should be the first thing in the file?  First you learn the process then, you learn how to do the dev bits?

But I'm just wareing my PM hat 😁

Reply:
Yes thats true, though I was looking at it as: heres how to setup to do development, then here is the process for developing and contributing it. But I guess some people need to only know the process for eg doc or dsign changes so yep I'll change this.